datadog-opentelemetry/src/lib.rs

```diff
@@ -165,10 +198,13 @@ pub(crate) mod propagation;
 pub(crate) mod sampling;
 
 mod ddtrace_transform;
+pub mod metrics_exporter;
+pub mod metrics_reader;
```

Comment:
Why are these modules publicly exposed?
The only item that  should probably be public is the `OtlpProtocol` enum, if we expose it in the config

Reply:
Good catch. I will make these modules internal to the crate. They should not be exposed to users